nasl/nasl_ssh.c

```diff
@@ -222,6 +222,8 @@ extern int lowest_socket;
  *
  * - @a scciphers SSH server-to-client ciphers.
  *
+ * - @a timeout Set a timeout for the connection in seconds
```

Comment:
```suggestion
 * - @a timeout Set a timeout for the connection in seconds.
```
As also asked in https://github.com/greenbone/openvas/pull/631/files#r541548160:

Is there also any information available which default SSH connection timeout libssh is used if no timeout (SSH_OPTIONS_TIMEOUT) was given?